site/src/atomics/spacing.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+These classes can be used to add spacing between elements.
+
+## Available sizes
+
+```Text
+`-none`
+`-xs`
+`-s`
+`-m`
+`-l`
+`-xl`
+`-xxl`
+`-xxxl`
+```
+
+## Margin and padding helpers
+
+For each of the spacing sizes, there are margin and padding helpers available:
+<br/><br/>
+
+| Size | Value         | Margin       | Padding       |
```

Comment:
To me, this list seems more at home on the tokens page, particularly since Atlas doesn't have table styles yet.

Reply:
Updated